src/Solitaire.java

```diff
@@ -0,0 +1,24 @@
+import GUI.GamePanel;
+
+import javax.swing.*;
+import java.io.IOException;
+
+public class Solitaire extends JFrame {
+
+    static protected GamePanel gamePanel;
+    public static final int gamePanelWidth = 1080;
+    public static final int gamePanelHeight = 1920;
+
+    public Solitaire() throws IOException {
+        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
Static constants should be statically imported

Furthermore, swing moved constants to dedicated interfaces (`JFrame` -> `WindowConstants`)

```suggestion
import java.io.IOException;

import static javax.swing.WindowConstants.EXIT_ON_CLOSE;

public class Solitaire extends JFrame {

    static protected GamePanel gamePanel;
    public static final int gamePanelWidth = 1080;
    public static final int gamePanelHeight = 1920;

    public Solitaire() throws IOException {
        setDefaultCloseOperation(EXIT_ON_CLOSE);
```